main/app.py

```diff
@@ -0,0 +1,352 @@
+
+
+        self.room_type = "O" or "L"
+
+        name = self.name = ""
+        personnel_type = self.personnel_type = ""
+        want_accommodation = self.want_accommodation = ""
+
+        self.offices = []
+        self.full_name = []
+        self.allocated_room = []
+
+        self.person_name = ""
+        self.person_type = ""
+        self.room_name = ""
+        self.people_in_room = []
+
+        self.person_name = []
+        self.person_type = ""
+
+        self.offices = []
+        self.housing = []
+
+        self.available_office = []
+        self.available_housing = []
+
+        self.office_allocations = []
+        self.living_alllocations = []
+        self.office_occupants = []
+        self.unallocated_offices = []
+        self.living_occupants = []
+        self.unallocated_houses = []
+        self.office = []
+        self.housing = []
+
+        self.living_occupied = 0
+        self.office_occupied = 0
+        file = self.file = " "
+        allocated = self.allocated = []
+
+        self.conn = sqlite3.connect("amity.sqlite")
+        self.conn.row_factory = sqlite3.Row
+        self.connect = self.conn.cursor()
+        self.connect.execute(
+            "CREATE TABLE IF NOT EXISTS Rooms(id INTEGER PRIMARY KEY AUTOINCREMENT, Name TEXT, type TEXT)")
+
+        self.connect.execute(
+            "CREATE TABLE IF NOT EXISTS Persons(id INTEGER PRIMARY KEY AUTOINCREMENT, Name TEXT, Personnel_type TEXT, want_accommodation TEXT, office_accommodation TEXT , living_accomodation TEXT)")
+
+    def create_rooms(self, room):
+        """Allows user to enter a list of room names specifying
+                whether office or living spaces"""
+        room_type = raw_input(
+            "Enter room type: \n O: Office space \n L: Living space: \n")
+
+        while room_type != "O" and room_type != "L":
+            room_type = raw_input(
+                "Try again. Enter Room Type:\n O: Office space \n L: Living space: \n")
+        for room_name in room:
+            if room_type.upper() == 'O':
+                self.rooms[room_type.upper()].append(room_name)
+            else:
+                self.rooms['L'].append(room_name)
+            self.connect.execute(
+                "INSERT INTO Rooms (Name, type) VALUES (?, ?)", [room_name, room_type])
+            self.conn.commit()
+        print('New rooms succesfully created')
+        return room
+
+    def office_space_count(self, off_name):
+        '''Keep track of offices allocated should be less than 6'''
+        off_name = str(off_name)
+        office_space = self.connect.execute(
+            "SELECT COUNT(*) AS office_occupants FROM Persons  WHERE Persons.office_accommodation = ?", [off_name]).fetchall()
+        self.office_occupied = office_space[0][0]
+        return self.office_occupied
+
+    def living_space_count(self, liv_name):
+        '''Keep track of living space allocated should be less than 4'''
+        # print(liv_name)
```

Comment:
**Code Quality: Nitpick**

Remember to clean out your codebase always. Don't leave commented out code in here.